### PR TITLE
Add loading spinner to button on bridge page

### LINKED ISSUE #34: Use loading spinner on button text when `usePrepareContractWrite.isLoading === true`
This applies to bridge page -- if you change the amount to bridge, you'll see the button become disabled, but with no indication that the prepare hook is loading

text could be replaced entirely with loading spinner

### PR DESCRIPTION
**Pull request by garyghayrat**
closes #34

When `usePrepareContractWrite.isLoading == true`, the transition would looks like this: 

https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/61768337/ae153362-770c-42e4-8288-b487fd20c3c7

### CODE REVIEW

**Comment by garyghayrat** [comment]
Current dependencies on/for this PR:
* main
  * **PR #58** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/58" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/58?utm_source=stack-comment).

File: pages/bridge.tsx [hunk 1]
```diff
@@ -213,14 +223,17 @@ const Bridge = () => {
                     type="button"
                     className="mt-5 mx-auto flex flex-row items-center rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600 disabled:bg-gray-300 disabled:cursor-not-allowed"
                     onClick={handleBridge}
-                    disabled={bridgeTarget === BridgeTarget.L2 ? !bridgeToL2 : !bridgeToL1}
+                    disabled={
+                      bridgeTarget === BridgeTarget.L2 ? bridgeToL2IsError : bridgeToL1IsError
```
Comment (wildmolasses) [inline pages/bridge.tsx:227]: What inspired this change?
can we gray out the button as disabled while the spinner is showing? I think that the earlier version of this line may accomplish that.

File: pages/bridge.tsx [hunk 2]
```diff
@@ -213,14 +223,17 @@ const Bridge = () => {
...
-                      (bridgeTarget === BridgeTarget.L1 && bridgeToL1IsLoading)) && (
-                      <div className="ml-2">
-                        <Spinner />
-                      </div>
+                    {(bridgeTarget === BridgeTarget.L2 && bridgeToL2IsLoading) ||
+                    (bridgeTarget === BridgeTarget.L1 && bridgeToL1IsLoading) ? (
+                      <Spinner />
+                    ) : (
+                      <span>
+                        Bridge to {target.chain.name} {bridgeToL2IsLoading.toString()}
```
Comment (wildmolasses) [inline pages/bridge.tsx:235]:
```suggestion
                        Bridge to {target.chain.name}
```

**Review by wildmolasses — CHANGES_REQUESTED** [review]
great start, requesting a couple small tweaks

File: pages/bridge.tsx (hunk 1 above)
Comment (garyghayrat) [reply pages/bridge.tsx]: umm, tbh I'm not sure anymore after the `easyWrite` changes, but I had to add extra params to the earlier version to gray it out. Check commit: [5a75f01](https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/58/commits/5a75f019426ca142d2bd83a49a7bb42e3e353c87)

File: pages/bridge.tsx (hunk 2 above)
Comment (garyghayrat) [reply pages/bridge.tsx]: My bad, removed in the same commit as above: [5a75f01](https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/58/commits/5a75f019426ca142d2bd83a49a7bb42e3e353c87)

**Review by wildmolasses — APPROVED** [review]
Works for me, thanks @garyghayrat !

**Pull request merged**